syntax/c.vim

```diff
@@ -1,7 +1,7 @@
 " Vim syntax file
 " Language:	C
-" Maintainer:	Bram Moolenaar <Bram@vim.org>
-" Last Change:	2016 Nov 18
+" Current Maintainer:	vim-jp (https://github.com/vim-jp/vim-cpp)
```

Comment:
We hope that diff(s) should be smallest, and should be only changes from official c.vim .